README.md

```diff
@@ -4,8 +4,8 @@ The following [.NET releases](../releases.md) are currently supported:
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 7](7.0/README.md) | [November, 2022](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7/) | [STS][policies] | [7.0.2][7.0.2] | May 14, 2024 |
-| [.NET 6](6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6/) | [LTS][policies] | [6.0.13][6.0.13]  | November 12, 2024 |
+| [.NET 7](../v7.0.0/README.md) | [November, 2022](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7/) | [STS](/release-policies.md) | [7.0.2][7.0.2] | May 14, 2024 |
```

Comment:
We probably meant to link to the files inside the release-notes directory here. @dcwhittaker?

Reply:
@mairaw Yes...I don't know when that changed since we don't touch that when we update the release-notes...but...I'll approve it :)